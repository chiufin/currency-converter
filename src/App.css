.App {
  background: #1e2f47;
  width: 100%;
  height: 100vh;
}

.App-logo {
  color: #fff;
}

.CurrencyBox-container {
  display: flex;

}

.CurrencyBox {
  margin: 0 10px;
  width: 300px;
  height: 250px;
  color: #fff;
}

.CurrencyBox-label {
  font-weight: 600;
}

.CurrencyBox-input{
  background: #293a51;
  height: 180px;
}

.CurrencyBox-convert-container{
  width: 30px;
  margin-top: 150px;
}

.CurrencyBox-convert-icon{
  width: 25px;
  height: 25px;
}

.CurrencyBox-input-dropdown-container{
  width: 100%;
  height: 50px;
  background: #2b465f;
  
  
}

.CurrencyBox-input-dropdown{
  width: 100%;
  height: 50px;
  background: #2b465f;
  border: none;
  font-size: 20px;
  color: #fff;
}

.CurrencyBox-input-text{
  width: 100%;
  height: 130px;
  padding-left: 20px;
  background: #293a51;
  border: none;
  font-size: 45px;
  color: #fff;
}

