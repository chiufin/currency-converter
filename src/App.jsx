import React from 'react';
import CurrencyBox from './CurrencyBox';
import './App.css';

function App() {
  return (
    <div className="App">
      <h1 className="App-logo">Currency Converter</h1>
      <CurrencyBox/>
    </div>
  );
}

export default App;
