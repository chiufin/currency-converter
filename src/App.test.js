import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('First React component test with Enzyme', () => {
   const wrapper = shallow(<App />);
   it('should match the snapshot', () => {
     expect(wrapper.html()).toMatchSnapshot();
   });
   it('renders without crashing', () => {
      shallow(<App />);
    });
});