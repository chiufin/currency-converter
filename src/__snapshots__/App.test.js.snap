// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`First React component test with Enzyme should match the snapshot 1`] = `"<div class=\\"App\\"><h1 class=\\"App-logo\\">Currency Converter</h1><div class=\\"CurrencyBox-container\\"><div class=\\"CurrencyBox\\"><p class=\\"CurrencyBox-label\\">1 USD = 1 EUR</p><div class=\\"CurrencyBox-input\\"><div class=\\"CurrencyBox-input-dropdown-container\\"><select class=\\"CurrencyBox-input-dropdown\\" name=\\"from\\"></select></div><input type=\\"text\\" class=\\"CurrencyBox-input-text\\" name=\\"from\\" value=\\"1\\"/></div></div><div class=\\"CurrencyBox-convert-container\\"><img class=\\"CurrencyBox-convert-icon\\" src=\\"convert_icon.png\\" alt=\\"covert icon\\"/></div><div class=\\"CurrencyBox\\"><p class=\\"CurrencyBox-label\\">1 EUR = 1 USD</p><div class=\\"CurrencyBox-input\\"><div class=\\"CurrencyBox-input-dropdown-container\\"><select class=\\"CurrencyBox-input-dropdown\\" name=\\"to\\"></select></div><input type=\\"text\\" class=\\"CurrencyBox-input-text\\" name=\\"to\\" value=\\"1\\"/></div></div></div></div>"`;
